requirements/requirements.txt

```diff
@@ -26,7 +26,6 @@ gunicorn==18.0
 lxml==3.4.4
 kafka-python==0.9.4
 mock==0.8.0
```

Comment:
the example you linked doesn't actually seem to be used in that file, no? are there others?